src/data/analysis/IOTM Reviews/Tearaway Pants.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "Tearaway Pants: Overview"
+date: "2024-08-21"
+slug: "2024-08-tearaway-pants"
+author: "Captain Scotch"
+section: "analysis"
+tag: "IOTM Overview"
+---
+WEIRD RAMBLING INTRO
+
+## General Summary
+
+Tearaway Pants are, well, pants. They're pants, folks. The pants are enchanted as follows:
+
+- + X Sleaze Damage (`X = level * 5`)
+- Damage Reduction: 10
+- So-So Resistance to All Elements (+2)
+- Combat Initiative +50%
+- Maximum HP +50
+- "Tear them away in combat!"
+
+The final bit of bluetext refers to a new combat skill, "Tear Away your Pants!" accessible whenever you happen to be wearing your tearaway pants. It is usable once per combat, staggering your foe and offering an additional effect depending on the phylum of the active monster:
+
+| Phylum                 | "Tear Away your Pants!" result             |
+| -------------------- | -------------------------------- |
+| Beasts, Bugs, and Penguins | 3-round stun |
+| Constellations, Elementals | Delevels the enemy |
+| Constructs, Slimes | Absolutely nothing |
+| Demons, Horrors, Undead, Weird | Physical damage to the enemy |
+| Dudes, Elves, Goblins, Hippies, Hobos, Humanoids, Orcs, Pirates | If the monster drops pickpocketable pants, you acquire them! Otherwise, it's a +15% bonus to item drop for the combat. |
+| Fish, Mer-Kin | Very tiny substat gains |
+| Plants | Gain 1 adventure, or no effect |
+
+In addition to the enchantments and the skill, the tearaway pants provide instant access to the Department of Shadowy Arts and Crafts for moxie classes. After all, Shifty's whole ask for guild membership was to steal your own pants, and it's very easy to steal your own tearaway pants!
+
+## Speedrun Applicability
+
+Let's start with the good news. This IOTM does two things that are really quite novel. First, it provides guild access to moxie classes without having to go off an ascension's critical path. Many people have been wondering if there will ever be a change where the mysticality and moxie guilds are placed on-path in the same way the muscle guild is; the fact that two classes get free access and it takes 5 annoying turns in a terrible zone for the other two has always felt a bit weird. This is better than we could've anticipated on that front, as it's actually *more* free than the muscle guild, since you'll get guild access without spending a single turn. That's neat! It should be helpful for newer players who still need to occasionally buy perms in-run. Beyond that, it doesn't really do much, but at least it does that.
```

Comment:
/takes 5 annoying turns in a terrible zone for the other four/ rather than /other two/ ?